src/main/java/com/infamous/combat_next/util/CombatUtil.java

```diff
@@ -218,16 +221,44 @@ private static AABB getSweepHitBox(Player player, ItemStack weapon) {
         return weapon.getSweepHitBox(player, player).move(xShift, 0.0, zShift);
     }
 
+    public static boolean isAttackAvailable(Player player, float partialTick) {
+        if (player.getAttackStrengthScale(partialTick) < 1.0F) {
+            LivingEntityAccessor livingEntityAccessor = (LivingEntityAccessor) player;
+            return PlayerCombat.cast(player).getMissedAttackRecovery() > 0 && livingEntityAccessor.getAttackStrengthTicker() + partialTick > PlayerCombat.cast(player).getMissedAttackRecovery();
```

Comment:
Does the missed attack recovery need to have an impact if it is 0 or less?

Reply:
not if it is less than 0, but maybe if it is 0

Reply:
oh wait, the default value missedAttackRecovery gets set to is 0, and it is reset to zero, so yes

Reply:
I made the default value -1 so it can be set as 0 without not working

Reply:
as in, default value for the variable used to know if an attack is a missed attack and how early it lets you, not the config default